app/kubernetes/views/applications/create/createApplicationController.js

```diff
@@ -458,7 +458,7 @@ class KubernetesCreateApplicationController {
   // * The data access policy is set to ISOLATED
   supportGlobalDeployment() {
     const hasFolders = this.formValues.PersistedFolders.length !== 0;
-    const hasRWOOnly = _.find(this.formValues.PersistedFolders, (item) => _.isEqual(item.StorageClass.AccessModes, ['RWO']));
+    const hasRWOOnly = _.find(this.formValues.PersistedFolders, (item) => item.StorageClass && _.isEqual(item.StorageClass.AccessModes, ['RWO']));
```

Comment:
this code ` item.StorageClass && _.isEqual(item.StorageClass.AccessModes, ['RWO'])` is repeated 3 times and then need to be updated 3 times, it would be preferred to extract it as a function.